.travis.yml

```diff
@@ -53,13 +54,11 @@ virtualenv:
 before_deploy:
   - cd $TRAVIS_BUILD_DIR
   - ./ci/travis/before_deploy.sh
-###  - cp $TRAVIS_BUILD_DIR/bindings/py/requirements.txt $TRAVIS_BUILD_DIR/bindings/py/dist/
-###  - mkdir -p $TRAVIS_BUILD_DIR/bindings/py/dist/include/nupic
-###  - cp ${TRAVIS_BUILD_DIR}/build/release/include/nupic/Version.hpp $TRAVIS_BUILD_DIR/bindings/py/dist/include/nupic
-###  - mkdir -p $TRAVIS_BUILD_DIR/bindings/py/dist/bin
-###  - cp ${TRAVIS_BUILD_DIR}/build/release/bin/py_region_test $TRAVIS_BUILD_DIR/bindings/py/dist/bin
-###  - mkdir -p $TRAVIS_BUILD_DIR/release
-###  - tar -zcvf $TRAVIS_BUILD_DIR/release/nupic_core-${TRAVIS_COMMIT}-${PLATFORM}64.tar.gz $TRAVIS_BUILD_DIR/bindings/py/dist
+#  - mkdir $BINARY_DIR/dist
+#  - cp $TRAVIS_BUILD_DIR/bindings/py/packaging/requirements.txt $BINARY_DIR/dist/
+  - mkdir -p $BINARY_DIR/dist/include/nupic
+  - cp ${TRAVIS_BUILD_DIR}/build/Release/include/nupic/Version.hpp $BINARY_DIR/dist/include/nupic
```

Comment:
> let's put the Version there in cmake install

Actually CMake puts it into the Release/include and setup.py copies it to dist/ so that line is redundant.